RichClipboard.package/RCClippingsList.class/instance/canCopy.st

```diff
@@ -0,0 +1,6 @@
+accessing
+canCopy
+	self copyAllowed: true
```

Comment:
At first glance, it wasn't clear to me that `canCopy` _sets_ the copyAllowed to true, instead of just returning true or false. I think typical squeak convention would be to set these values in a method with a colon `:`. We could just replace all instances of this with `copyAllowed: true` — to me they say the same thing and it saves extra code. Same goes for `cantCopy`.